README.md

```diff
@@ -1,75 +1,132 @@
-## Scripts
+# DLT (DualLayerToken) Standard 🛠️
 
-This repository includes the following scripts in the `package.json` file:
+Welcome to the DualLayerToken (DLT) Standard, proposed under EIP-6960! 🎉
```

Comment:
**DualLayerToken** I didn't see any concatenated topics before, does it need spaces ?